lightly/api/api_workflow_selection.py

```diff
@@ -52,22 +52,20 @@ def upload_scores(self, al_scores: Dict[str, np.ndarray], query_tag_id: str = No
                 tag_id=query_tag_id,
             )
 
-    def sampling(self, sampler_config: SamplerConfig, preselected_tag_id: str = None, query_tag_id: str = None) \
+    def selection(self, selection_config: SelectionConfig, preselected_tag_id: str = None, query_tag_id: str = None) \
```

Comment:
this is a breaking change. If we really want to do this now we should keep the old method and raise a deprecation warnign.

Reply:
I kept the old method (just calling the new one) and added a PendingDeprecationWarning. A unittest checks for it.